tensorflow/python/kernel_tests/shape_ops_test.py

```diff
@@ -411,14 +411,17 @@ def testScalar(self):
       self.assertEqual(7, result)
 
   def testSimple(self):
-    with self.test_session():
-      inp = np.random.rand(4, 1).astype(np.float32)
-      a = constant_op.constant(inp)
-      tiled = array_ops.tile(a, [1, 4])
-      result = tiled.eval()
-    self.assertEqual(result.shape, (4, 4))
-    self.assertEqual([4, 4], tiled.get_shape())
-    self.assertTrue((result == np.tile(inp, (1, 4))).all())
+    for use_gpu in False, True:
```

Comment:
You can just do use_gpu=True; if a GPU is not available the test will run on CPU, and our test infrastructure runs all tests on CPU and GPU machines.

Reply:
Thanks @vrv. Done.